public/sanda/practice/sanda4.5.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Practice 4-5</title>
+</head>
+<body>
+
+    <script>
+        function sayHello(name) {
+            console.log(`Hello, ${name}!`);
+        }
+        sayHello("Tanya");
+
+        /*Random number */
+        function generateRandomInt() {
+            let n = Math.floor(Math.random() * Math.floor(20));
+            console.log(`Random integer: ${n}`);
+            return n;
+        }
+        generateRandomInt();
+
+        //Random min/max
+        function generateRandomIntInRange(min, max) {
+            let n = Math.floor(Math.random() * (max - min) + min) ;
+            console.log(`Random integer in range ${min} and ${max}: ${n}`);
+            return n;
+        }
+        generateRandomIntInRange(56, 72);
+
+        /* is Even*/
+        function isEven(value) {
+            if (value % 2 === 0) {
+                return true;
+            } else {
+                return false;
+            }
+        }
+        let m =  generateRandomInt();
+        if (isEven(m)) {
+            console.log(`${m} is even`);
+        }
+        else {
+            console.log(`${m} is odd`);
+        }
+
+        /* List of districts */
+        function getTheBestDistrict() {
+            let districts = [
+                'Ciocana',
+                'Rascani',
+                'Centru',
+                'Buiucani',
+                'Botanica'
+            ];
+            return districts[generateRandomIntInRange(0, 5)];
```

Comment:
Please, use length of your `districts` variable instead of `5` here.

Reply:
Please, provide more details, because I want to end this quarantine with pull request approved.

Reply:
"5" is a hardcoded value here. In this case it means length of your array. But if you add one more item to the array, the code will stop working correctly until you change "5" to "6" manually. This is not flexible.
Knowing how to get length of an array in Javascript, please, avoid hardcode.